guides/doc-Hammer_CLI/topics/con_introduction-to-hammer.adoc

```diff
@@ -0,0 +1,457 @@
+----
++
+* To end the session, enter the following command:
++
+----
+# hammer auth logout
+----
+
+[[sect-CLI_Guide-Authentication-Hammer_Configuration_File]]
+=== Hammer configuration file
+
+If you ran the {Project} installation with `--foreman-initial-admin-username` and `--foreman-initial-admin-password` options, credentials you entered are stored in the `~/.hammer/cli.modules.d/foreman.yml` configuration file, and hammer does not prompt for your credentials.
+
+You can also add your credentials to the `~/.hammer/cli.modules.d/foreman.yml` configuration file manually:
+
+[options="nowrap", subs="+quotes,attributes"]
+----
+:foreman:
+ :username: '_username_'
+ :password: '_password_'
+----
+
+[IMPORTANT]
+====
+Use only spaces for indentation in hammer configuration files.
+Do not use tabs for indentation in hammer configuration files.
+====
+
+[[sect-CLI_Guide-Authentication-Command_Line]]
+=== Command line
+
+If you do not have your {Project} credentials saved in the `~/.hammer/cli.modules.d/foreman.yml` configuration file, hammer prompts you for them each time you enter a command.
+You can specify your credentials when executing a command as follows:
+
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ hammer -u _username_ -p _password_ _subcommands_
+----
+
+[NOTE]
+====
+Examples in this guide assume that you have saved credentials in the configuration file, or are using a hammer authentication session.
+====
+
+[[sect-CLI_Guide-Standalone_Use_of_Hammer]]
+== Using standalone Hammer
+
+You can install `hammer` on a host running {RHEL} 8 that has no {ProjectServer} installed, and use it to connect the host to a remote {Project}.
+
+.Prerequisites
+* Ensure that you register the host to {ProjectServer} or {SmartProxyServer}.
+For more information, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.
+ifdef::katello,orcharhino,satellite[]
+* Ensure that you synchronize the following repositories on {ProjectServer}:
+ifdef::satellite[]
+** {RepoRHEL8BaseOS}
+** {RepoRHEL8AppStream}
+** {RepoRHEL8ServerSatelliteUtils}
+endif::[]
+ifndef::satellite[]
+** {EL} 8 BaseOS
+** {EL} 8 AppStream
+** ??? what is the Utils alternative for Foreman/Katello?
+endif::[]
+
++
+For more information, see {ContentManagementDocURL}Synchronizing_Repositories_content-management[Synchronizing Repositories] in _{ContentManagementDocTitle}_.
+endif::katello,orcharhino,satellite[]
+
+.Procedure
+. Enable the required repositories on the host:
++
+ifdef::satellite[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL8BaseOS} \
+--enable={RepoRHEL8AppStream} \
+--enable={RepoRHEL8ServerSatelliteUtils}
+----
+endif::[]
+ifndef::satellite[]
+??? is there an alternative for Foreman/Katello?
+endif::[]
+. Enable the {Project} Utils module:
++
+[options="nowrap" subs="verbatim,quotes,attributes"]
+----
+# dnf module enable satellite-utils:el8
+----
```

Comment:
I need help here. :)

Reply:
For upstream it's the exact same repository as the server so you should be able to reuse that module